### DISCUSSION TITLE
How to open multiple Notion instances?

### DISCUSSION THREAD

**Discussion by lsps9150414** [ttscoff/bunch-docs#252] [Q&A]
I want to open multiple Notion instances each for a different page. It used to work with:
```bash
$ open -n \
  notion://some-page-url \
  notion://another-page-url
```
but it does not work anymore. I've recently switched to a new Macbook with M1 Pro chip. Not sure if it's related.

**Comment by ttscoff** [comment]
I don't have Notion installed, but your syntax works fine using 
x-nvultra url handlers (on an M1). I'm assuming you're not expecting 2 
different instances to appear from the single command, right? Is nothing 
happening? Does the command work correctly from Terminal?

Go to Help->Set Log Level and set it to debug, then use Help->Open Log 
while running the Bunch to see if you get any useful errors.
